### ISSUE TITLE
Any plans to support Serial Slave Operation?

### ISSUE THREAD

**Issue by bravecorvus** [afch/NixeTubesShieldNCS314#3]
So I know currently you guys don't support Slave via Serial USB, but do you have any plans to add this in the future? ([like this](https://github.com/afch/Slave-Nixie-Clock)

**Comment by afch** [comment]
We did not plan to do this in the near future.
But we are sure that it will not be difficult at all to transfer this functional from the old version.
In addition, we have an experimental adapter for connecting our shields to the RaspberryPI.
![dsc01003](https://user-images.githubusercontent.com/2285746/33023499-0b14081a-ce11-11e7-8ca9-2474e01ab171.jpg)
![dsc01020_](https://user-images.githubusercontent.com/2285746/33023500-0b387312-ce11-11e7-99e2-8ee12835e8c4.jpg)
![dsc01019](https://user-images.githubusercontent.com/2285746/33023595-5882d5f4-ce11-11e7-962f-4b786f4789b2.jpg)

**Comment by gubagu** [comment]
Why not to esp8266 instead of raspberry? RPI is such an overkill for a clock. Though it could be a bit of a hassle if you the project uses a lot pwm pins. esp8266 has only one analog pin ...

**Comment by bravecorvus** [comment]
True, but essentially in already using a Pi for the alarm functionality for [my project](https://hackaday.io/project/15048-prometheus-alarm-clock). So I thought I might as well try to utilize NTP to control the nixie displays as well.
